gcloud-java-datastore/README.md

```diff
@@ -36,6 +48,8 @@ Cloud Datastore for your project.
 See the ``gcloud-java`` API [datastore documentation][datastore-api] to learn how to interact
 with the Cloud Datastore using this Client Library.
 
+Here is a code snippet showing a simple usage example from within Compute/App Engine.  Note that you must [supply credentials](https://github.com/GoogleCloudPlatform/gcloud-java#authentication) if running this snippet elsewhere.
```

Comment:
actually, if you run Compute/App Engine you don't also need to provide projectId. (if we remove it from snippet, which should mention that it should be added in "elsewhere").